frontend/components/MainContent/_styles.scss

```diff
@@ -19,14 +19,16 @@
   }
 }
 
-@media (max-width: ($break-xs - 1)) {
+@media (min-width: $break-md) {
   .main-content {
     padding: $pad-large;
   }
 }
 
-@media (max-width: ($break-mobile-md - 1)) {
+@media (min-width: $break-lg) {
   .main-content {
-    padding: 20px;
+    padding: $pad-xlarge;
   }
 }
+
+
```

Comment:
This is a mobile-first progressive enhancement pattern that I've found to work really well as complexity builds up. 

The idea is the default styles apply to your smallest supported screen size, and as the screen grows you progressively add more style overrides using `min-width`. This ends up reading more simple because you don't have to think through the defined styles being progressively removed using `max-width`.